src/main/java/org/example/inspector/SongInspector.java

```diff
@@ -0,0 +1,7 @@
+package org.example.inspector;
+
+import java.io.File;
+
+interface SongInspector {
```

Comment:
It's really not an "inspector" as "inspect" means it doesn't change anything. It's probably `MediaFileNameRenamer` as the repo is called "renamer".